assets/scss/gravity-forms.scss

```diff
@@ -468,3 +468,59 @@
   color      : $color_nhsuk-black;
   padding    : nhsuk-spacing(1);
 }
+
+.gf-all-fields { // style up the all-fields summary to look like it should. This is a sleight of hand.
+  @extend .nhsuk-summary-list;
+  li {
+	&.field-type-section {
+	  background-color: $color_nhsuk-white; // !important required to over ride plugin genertaed inline styling.
```

Comment:
Is this comment needed? I don't see an important tag here

Reply:
good spot :)